### DISCUSSION TITLE
app tray

### DISCUSSION THREAD

**Discussion by fh222333A** [pear-devs/pear-desktop#98] [Ideas]
Is possible to change the app tray left click to pause/ play the music instead of opining the app??
thx :)

**Comment by th-ch** [comment]
Hey @fh222333A thanks for the suggestion! It makes sense, it will be tracked in https://github.com/th-ch/youtube-music/issues/119

**Reply by th-ch** [reply]
Hey @fh222333A, [latest version 1.9.0](https://github.com/th-ch/youtube-music/releases/tag/v1.9.0) adds an option to play/pause on tray click, feel free to give it a try!

**Comment by inson1** [comment]
@fh222333A May you close this discussion, if it is resolved? :)
